packages/currency/src/native.ts

```diff
@@ -123,6 +123,18 @@ export const nativeCurrencies: Record<RequestLogicTypes.CURRENCY.ETH, NativeEthC
       name: 'Tomb',
       network: 'tombchain',
     },
+    {
+      symbol: 'MNT',
+      decimals: 18,
+      name: 'Mantle',
+      network: 'mantle',
+    },
+    {
+      symbol: 'MNT',
```

Comment:
I think symbols must be unique, and should not be confused with their mainnet counterpart. 
Check what we do for other testnets